scripts/integration.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+set -e
+
+git clone https://github.com/zeta-chain/template
+
+cd template
+
+yarn
+
+yarn link @zetachain/toolkit
+
+npx hardhat balances
+
+npx hardhat fees
+
+npx hardhat account --save
+
+if [[ ! -f .env ]]; then
+    echo ".env file was not created."
+    exit 1
+fi
+
+if grep -E "^PRIVATE_KEY=[a-fA-F0-9]{64}$" .env; then
+    echo "The .env file contains a valid PRIVATE_KEY entry without the 0x prefix."
+    rm .env
```

Comment:
You mean why have `rm .env`? The reason is in this section of the test I'm testing the key generation functionality, but I don't need the actual key for the rest of the test. I'm using a `PRIVATE_KEY` env variable stored in GitHub secrets, which has enough tokens (both native and ZETA) to deploy and test contracts. At some point I'll add faucet to the test, but I'll probably still keep using the evn variable, because I need native gETH, for example, which I can't get from a faucet.